frontend/public/components/utils/_resource-log.scss

```diff
@@ -0,0 +1,13 @@
+.resource-log {
+  height: 100%;
+  width: 100%;
+  &.log-console--fullscreen {
```

Comment:
I don't think this needs to be nested under .resource-log

Reply:
Shouldn't this also be `&.resource-log--fullscreen?

Reply:
Hi, fixed, still learning BEM, thanks reviewing

Reply:
... or if not nested, `.resource-log.resource-log--fullscreen`